doc/releases/release-notes-3.7.rst

```diff
@@ -300,6 +300,24 @@ Libraries / Subsystems
       registered to set the device attributes that are sent to the hawkBit server. Use the
       :c:func:`hawkbit_set_custom_data_cb` function to register the callback.
 
+    * :c:func:`hawkbit_autohandler` now takes one argument. If the argument is set to true, the
+      autohandler will reshedule itself after running. If the argument is set to false, the
+      autohandler will not reshedule itself. Both variants are sheduled independent of each other.
+      The autohandler always runs in the system workqueue.
+
+    * With :c:func:`hawkbit_autohandler_wait` you can wait for the autohandler to finish. This
+      function will block until the autohandler is finished.The ``event`` parameter can be passed
+      ``UINT32_MAX`` to wait for the autohandler to finish one run, or passed :c:macro:`BIT`
```

Comment:
Nit
```suggestion
      function will block until the autohandler is finished. The ``event`` parameter can be set to
      ``UINT32_MAX`` to wait for the autohandler to finish one run, or :c:macro:`BIT`
```
Anyhow, the whole "This  function will block until the autohandler is finished" seems a bit redundant.